index.html

```diff
@@ -4286,336 +5118,508 @@ <h3 id="8_A_4">8.A.4 Example</h3>
                     <strong>Request</strong>
                 </p>
                 <pre>
-GET https://api.myparcel.nl/delivery_options?cc=NL&amp;postal_code=2132WT&amp;number=66&amp;carrier=1&amp;cutoff_time=16:00:00&amp;dropoff_days=1;2;3;4;5;6&amp;monday_delivery=1&amp;dropoff_delay=0&amp;deliverydays_window=2&amp;exclude_delivery_type=1  HTTP/1.1
+GET https://api.myparcel.nl/delivery_options/?platform=myparcel&carrier=postnl&cc=nl&number=66&postal_code=2132WT&cutoff_time=16:00:00&dropoff_days=1;2;3;4;5;6&monday_delivery=1&dropoff_delay=0&deliverydays_window=2&exclude_delivery_type=1   HTTP/1.1

```

Comment:
```suggestion
GET https://api.myparcel.nl/delivery_options?platform=myparcel&carrier=postnl&cc=nl&number=66&postal_code=2132WT&cutoff_time=16:00:00&dropoff_days=1;2;3;4;5;6&monday_delivery=1&dropoff_delay=0&deliverydays_window=2&exclude_delivery_type=1   HTTP/1.1

```
Waarom met een `/`? Werkt het anders niet?

Reply:
Het werkt allebei